src/guiFormSpecMenu.cpp

```diff
@@ -2717,6 +2717,19 @@ void GUIFormSpecMenu::drawMenu()
 	drawSelectedItem();
 
 	skin->setFont(old_font);
+
+	const std::string &tp_path = g_settings->get("texture_path");
+	std::string cursor_img_path = tp_path + "/cursor.png";
+
+	if (fs::PathExists(cursor_img_path)) {
+		m_device->getCursorControl()->setVisible(false);
+		video::ITexture *cursor_img = driver->getTexture(cursor_img_path.c_str());
```

Comment:
also this texture is loaded from disk at each draw, this will be a huge performance penalty. Please load texture and setting in guiFormSpecMenu constructor & drop image in destructor.